src/content/index.mdx

```diff
@@ -1,25 +1,18 @@
 ---
 layout: page
-title: 'The Guardian Engineering'
+title: ''
```

Comment:
I think we should still set this as the title (on the tab)

Reply:
The tab title comes from the site metadata which is set separately. This would only be used for anything that references `frontmatter`. I've added it back in because I don't like the empty string, but I think it's not actually doing anything.

![image](https://user-images.githubusercontent.com/9122944/97290831-01f97a80-1841-11eb-99f0-b240f50348da.png)
This shows the tab with the title frontmatter set.